### PR TITLE
Publish from Travis CI after successful builds

### PR DESCRIPTION
**Pull request by dialelo**
@raulraja @rafaparadela please take a look.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -6,4 +6,8 @@ jdk:
 script:
   - git clone https://github.com/scala-exercises/site.git site
   - pushd site; sbt 'definitions/publishLocal' 'runtime/publishLocal' 'compiler/publishLocal' 'sbt-exercise/publishLocal' ; popd
-  - sbt test
\ No newline at end of file
+  - sbt test
+after_success:
+  - echo $KEYS_PASSPHRASE | gpg --passphrase-fd 0 keys.tar.gpg
+  - tar xfv keys.tar.gz
+  - sbt publishSigned
```
Comment (raulraja) [inline .travis.yml]: Would this only happen for master builds? We don't want snapshots when people issue PRs and Travis kicks in.

Comment (dialelo) [inline .travis.yml]: Good point, I need to update this to only release on the master branch.

**Comment by dialelo** [comment]
@raulraja modified the travis config to only publish from the `master` branch.

**Comment by raulraja** [comment]
LGTM!

**Pull request merged**